### ISSUE TITLE
Create a Docker image to run the whole project

### ISSUE THREAD

**Issue by jabrena** [TheFirstClassCitizen/microservices-2018#10]
The docker image should run:

* Kafka Bus
* Spring Cloud Core components
* Microservices

https://spring.io/guides/gs/spring-boot-docker/
http://www.baeldung.com/dockerizing-spring-boot-application
http://containertutorials.com/docker-compose/spring-boot-app.html
https://dzone.com/articles/running-spring-boot-in-a-docker-container

**Comment by juan-medina** [comment]
what you mean the whole? do think in run the whole thing in a single docker, each component need a docker, then they could scale and deploy independently.

If not you are just doing a distribute monolith: https://www.infoq.com/news/2016/02/services-distributed-monolith

**Comment by jabrena** [comment]
No, the idea is to run every component in a different container.

**Comment by jabrena** [comment]
http://accordance.github.io/microservice-dojo/kata2/dockerizing_application.html
